attributes/default.rb

```diff
@@ -4,6 +4,11 @@
 default['newrelic-infra']['agent_action'] = 'install'
 default['newrelic-infra']['agent_version'] = nil
 
+default['newrelic-infra']['yum'] = {}
+default['newrelic-infra']['yum']['baseurl'] = nil
+default['newrelic-infra']['yum']['gpgkey'] = nil
+default['newrelic-infra']['yum']['repo_gpgcheck'] = true
```

Comment:
This should just be `gpgcheck` to match the attributes in https://docs.chef.io/resource_yum_repository.html (we aren't using that resource directly, but I would expect the names to match.)